src/pages/about-us.js

```diff
@@ -12,6 +13,7 @@ const AboutUs = ({ data: { contentfulAboutUsPage: content } }) => {
     <Layout>
       <Head page={content} />
       <Subsidiaries title={yldGroupTitle} subsidiaries={subsidiaries} />
+      <Teams title={content.teamSectionTitle} teams={content.teams} />
```

Comment:
maybe destructure these on line 10, just for consistency?

Reply:
done